<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_gerenciarVenda</class>
 <widget class="QDialog" name="window_gerenciarVenda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciar Vendas</string>
  </property>
  <widget class="QDateEdit" name="deInicial">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="deFinal">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gerarPdf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Gerar Relat√≥rio PDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuscarVendas">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Inicial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Final</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twVendas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>571</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="twProdutosVendas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>571</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_todasVendas">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostrar todas  Vendas</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
